components/list/list.js

```diff
@@ -87,11 +93,26 @@ class List extends SelectionMixin(LitElement) {
 			}, 0);
 
 		});
+
```

Comment:
The `d2l-list-header` needs to know when the list is scrolled and the header is sticking so that it can show/hide the shadow.  The way this is accomplished is: the list renders a sentinel element, and we use `IntersectionObserver` to detect when it intersects the view-port. The list then sets a private property on the `d2l-list-header` indicating that state.

Reply:
Haha "sentinel" sounds so ominous, love it.